data/drak/drak ships.txt

```diff
@@ -115,6 +115,27 @@ ship "Archon" "Archon (Cloaked)"
 		"cloak" .04
 	"final explode" "archon c teleport"
 
+ship "Archon" "Archon (Hyperdrive Only)"
```

Comment:
Since this has two differences from the base variant, being both HD-only and cloak-capable, I'd maybe call it something like `"Archon (Hyperdrive Cloaked)"` or similar, in case someone wants to add a non-cloaked HD-only Archon in future.